cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -984,9 +976,11 @@ flags should be captured within MPAS CMake files.
   <LAPACK_LIBDIR>/software/tools/compilers/intel_2017/mkl/lib/intel64</LAPACK_LIBDIR>
   <LDFLAGS>
     <append compile_threaded="TRUE"> -fopenmp </append>
-    <append COMP_CLASS="cpl"> -L$NETCDF_PATH/lib -Wl,-rpath=$NETCDF_PATH/lib -lnetcdff -lnetcdf </append>
     <!-- <append COMP_CLASS="cpl"> -L$ENV{CLM_PFLOTRAN_SOURCE_DIR} -lpflotran $ENV{PETSC_LIB} </append> -->
   </LDFLAGS>
+  <SLIBS>
+    <append> -L$NETCDF_PATH/lib -Wl,-rpath=$NETCDF_PATH/lib -lnetcdff -lnetcdf </append>
```

Comment:
Is there a reason to remove `COMP_CLASS="cpl"` in the original append line here?

Reply:
Whoever did that just didn't understand our build system. We only do linking once for e3sm.exe, so it's pointless/redundant to limit LDFLAGS to the coupler.

Reply:
OK, thanks for the information.